asr1k_neutron_l3/models/netconf_yang/nat.py

```diff
@@ -126,26 +129,29 @@ def __init__(self, **kwargs):
         self.vrf = self.id
 
     def to_dict(self, context):
-        pool = OrderedDict()
-        pool[NATConstants.ID] = self.id
-
-        pool[NATConstants.START_ADDRESS] = self.start_address
-        pool[NATConstants.END_ADDRESS] = self.end_address
-        pool[NATConstants.NETMASK] = self.netmask
-
-        result = OrderedDict()
-        result[NATConstants.POOL] = pool
+        result = {
+            NATConstants.POOL: {
```

Comment:
We could do that but I think it would also change the yang options. To be honest, when I saw that we already had NATPooling enabled and that this was apparently even partly working, I just went with what was there. I removed some of the OrderedDicts, as I found them to be unnecessary and harder to read, but stuck with everything else.